ansible/roles/openshift_setup/tasks/main.yml

```diff
@@ -287,8 +287,10 @@
       msg: "Looking at oc cluster up command:  '{{ oc_cluster_up_cmd }}'"
 
   - name: Run oc cluster up to start the cluster
-    shell: "{{ oc_cluster_up_cmd }}"
+    shell: "{{ oc_cmd }} cluster down && {{ oc_cluster_up_cmd }}"
```

Comment:
Am I missing something? It already knows if it failed by the exit code of the oc cluster up. Why the additional complexity?